aiida_fleur/workflows/relax.py

```diff
@@ -340,12 +340,18 @@ def condition(self):
 
         largest_now = self.ctx.forces[-1]
 
+        # get force mixing (straight or BFGS) setting
+        # defaults to stright mixing if not set in scf.wf_parameters
+        if 'wf_parameters' in self.inputs.scf.wf_parameters:
+            force_strmix = self.inputs.scf.wf_parameters['force_dict']['forcemix'] == 'straight'
```

Comment:
```suggestion
        if 'wf_parameters' in self.inputs.scf:
            force_dict = self.inputs.scf.wf_parameters.get_dict().get('force_dict',{})
            force_strmix = force_dict.get('forcemix', 'straight') == 'straight'
```
Unfortunately i think this fix does not work in every circumstance. Since the `wf_parameters` input has defaults that are replaced inside the scf workchain, less assumptions can be made about the presence of these keys.